erpnext/stock/doctype/item/item.js

```diff
@@ -720,6 +732,15 @@ $.extend(erpnext.item, {
 			});
 		}
 
+		if (frm.doc.image) {
+			fields.push({
+				fieldtype: "Check",
+				label: __("Create a variant with the template image."),
+				fieldname: "use_same_image",
```

Comment:
Please change the variable name use_same_image -> use_template_image
Also make changes in py file accordingly

Reply:
Done 👍